charts/postiz/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: postiz-app
 description: A Social Media Scheduling App 
 type: application
-version: 1.0.1
+version: 1.0.2
```

Comment:
The trailing spaces on this line is causing the CI lint failure.

Reply:
I removed the trailing space! Thanks